cves/kernel/CVE-2016-8630.yml

```diff
@@ -219,8 +245,10 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: x86
```

Comment:
I don't think x86 is a subsystem. I think `kvm` fits better here. Or you could put both.